internal/api/errorcodes.go

```diff
@@ -78,4 +78,8 @@ const (
 	ErrorCodeHookPayloadOverSizeLimit          ErrorCode = "hook_payload_over_size_limit"
 	ErrorCodeHookPayloadUnknownSize            ErrorCode = "hook_payload_unknown_size"
 	ErrorCodeRequestTimeout                    ErrorCode = "request_timeout"
+	ErrorCodeMFASMSEnrollDisabled              ErrorCode = "mfa_sms_enroll_not_enabled"
+	ErrorCodeMFASMSVerifyDisabled              ErrorCode = "mfa_sms_verify_not_enabled"
+	ErrorCodeMFATOTPEnrollDisabled             ErrorCode = "mfa_totp_enroll_not_enabled"
+	ErrorCodeMFATOTPVerifyDisabled             ErrorCode = "mfa_totp_verify_not_enabled"
```

Comment:
can consider combining them into 2 distinct groups since they describe the same class of errors?
1. `mfa_enroll_not_enabled` 
2. `mfa_verify_not_enabled`

Reply:
Prev discussion [here]:(https://github.com/supabase/auth/pull/1668#discussion_r1689470203)

Works too @hf wdyt?

Reply:
ah i see, hmm i'm ok either way too - doesn't really hurt to have more error codes i guess, just note that we'll be adding 2 more codes (from webauthn too)

Reply:
Er will go with two codes for now but we can revisit if needed before we launch